app/_components/inputs/tests/Modules.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { act, render, screen } from '@testing-library/react';
+import { useForm } from 'react-hook-form';
+import { ChakraProvider, FormControl, FormLabel } from '@chakra-ui/react';
+import selectEvent from 'react-select-event';
+import { StationForm } from '@/app/_types/forms';
+import ModulesField from '../Modules';
+
+const Component = ({ isMulti = false }: { isMulti?: boolean }) => {
+  const { control } = useForm<StationForm>();
+
+  return (
+    <ChakraProvider>
+      <ModulesField control={control} isMulti={isMulti} />
+    </ChakraProvider>
+  );
+};
+
+const MODULES = [
+  '1I Detailed Surface Scanner',
+  '5D Fighter Hangar',
+  '6D Fighter Hangar',
+  '7D Fighter Hangar',
+  '2A Frame Shift Drive',
```

Comment:
Do we need all these modules and ships or a few of these will do?

Reply:
Not essential, just copy pasted my dummy data list - have trimmed them down to a few entries